src/components/SelectListForm.jsx

```diff
@@ -3,17 +3,19 @@ import { useState } from 'react';
 
 export default function SelectListForm({
 	data,
-	listPath,
 	areListsLoading,
+	listName,
```

Comment:
You need to make sure this prop is actually being passed in. Where is it being passed in from?